core/pom.xml

```diff
@@ -162,7 +162,7 @@
 
         <!-- Swagger -->
         <dependency>
-            <groupId>io.swagger</groupId>
+            <groupId>io.swagger.core.v3</groupId>
```

Comment:
Updating swagger annotations

Reply:
Why do we need this updated for this PR?
Consider doing this as a separate PR.